docs/moment/-project-status/01-recommendations.md

```diff
@@ -42,3 +42,11 @@ If you are familiar with [`java.time`](https://docs.oracle.com/javase/8/docs/api
 
 - Locales: Custom data files via add-on module
 - Time Zones: Custom data files via add-on module
+
+### No Library
+
+JavaScript has always had a `Date` object.  Its not perfect, but it has evolved somewhat over time.  Parsing strings is still problematic, so be careful there.
```

Comment:
What do you mean "Parsing strings is still problematic" ? Could you elaborate a little?

Reply:
Parsing Strings with the Date constructor and Date.parse has a few issues:
- It switches between the local timezone and UTC in a surprising way (both between JS versions and within ES6+ depending on which components exist in the date string)
- Strings which don't conform to a particular subset of ISO 8601 have implementation-defined syntax and semantics. This is a frequent source of browser compatibility bugs for JS developers, and is a won't-fix issue filed (and frequently duped) against every browser (because everyone is waiting for a standard, but no one is doing the work to build that standard)

The current text seems good, but I guess it's hard to understand what to do with "be careful". I would suggest that you specifically recommend against using Date string parsing here, and to instead use the number-based constructor, if you have everyone's attention in this doc.